opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/Configuration.java

```diff
@@ -580,7 +580,7 @@ public Configuration() {
         setHandleHistoryOfRenamedFiles(false);
         setHistoryBasedReindex(true);
         setHistoryCache(true);
-        setHistoryEnabled(true);
+        setHistoryEnabled(false);
```

Comment:
done to avoid `cfg.setHistoryEnabled(false);` in the `Indexer` which causes confusing warning messages about discrepancy between history and history based reindex.